### PR TITLE
PR changes

### PR DESCRIPTION
**Pull request by karthikeyan7010**

### CODE REVIEW

File: config/config.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Object.assign(global, {
+  BASE_URL: 'http://local.magento.com/',
+  firstname:'Karthick',
```
Comment (Vigneshmuthusamy) [inline config/config.js:3]: @karthikeyan7010 Registration fields are inputted as example in feature

File: features/NewCustomerRegistration.feature [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    And clicks on Create an Account button
+
+Scenario: Empty Customer Registration Authentication (Negative scenario)
+    Given user is in customer registration page
+    When the user clicks on create an account button
+    Then the user should see 5 error message "This is a required field"
+
+Scenario: Emailid  validation for Customer Registration Authentication (Negative scenario)
+    When I fill the login form with "<emailid>"
+    Then the user should see "Please enter a valid email address Please enter a valid email address Ex: johndoe@domain.com"
```
Comment (Vigneshmuthusamy) [inline features/NewCustomerRegistration.feature:15]: @karthikeyan7010 There is typo in this sentence

File: features/NewCustomerRegistration.feature [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        Examples:
+                | emailid | 
+                | karthikeyan3071997gmail.com |
+                | karthikeyan3071997@gmail|
+                | karthikeyan3071997.com|
+
+Scenario: Password  validation for Customer Registration Authentication (Negative scenario)
+    When User enters password with invalid format "<password>"
+    Then the user should see "Minimum of different classes of characters in password is 3. Classes of characters: Lower Case, Upper Case, Digits, Special Characters" message
```
Comment (Vigneshmuthusamy) [inline features/NewCustomerRegistration.feature:24]: @karthikeyan7010 user should see error message, "...."

File: features/NewCustomerRegistration.feature [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+            | karthick | 
+            | 12345678 |
+            | karthick1234 |
+
+
+
+Scenario: valid Registration Authentication (Postive scenario)
+    Given The user enters valid credentials in the required fields for customer registration
+    When click create an account button
+    Then the user is created and redirected to My Account page
```
Comment (Vigneshmuthusamy) [inline features/NewCustomerRegistration.feature:36]: @karthikeyan7010 Use the example here for adding multiple user registrations and validate if the user email is what we have given during registration in my account page

File: features/registercustomer.feature [hunk 5]
```diff
@@ -0,0 +1,43 @@
+Feature: Order placement using Register customer
```
Comment (Vigneshmuthusamy) [inline features/registercustomer.feature:1]: @karthikeyan7010 I told you several times to name features in sync with the feature we are trying to test. I see feature name as registercustomer

File: features/registercustomer.feature [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+Background: Order placement using Register customer
```
Comment (Vigneshmuthusamy) [inline features/registercustomer.feature:3]: @karthikeyan7010 Login featue and order placement should be separate features.

File: features/registercustomer.feature [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    And click a sigin button
+    Then the user should see "Incorrect CAPTCHA"
+        Examples:
+            | password | 
+            | karthick | 
+            | 12345678 |
+            | karthick1234 |
+
+Scenario: Order placement using Register customer (postive scenario)
+    Given the user should navigated to Sigin page
```
Comment (Vigneshmuthusamy) [inline features/registercustomer.feature:33]: @karthikeyan7010 Again same mistake. Refer all my previous comments and fix this scenario

File: page-objects/NewCustomerRegistration-page.js [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+    await page.click('button:has-text("Create an Account")');
+}
+
+async passworderrormessage() {
+    const passworderrormessage =  await page.isVisible('#password-error');
+    return expect(passworderrormessage).to.equal(true);
+    
+}
+
+    async validcredentials() {
```
Comment (Vigneshmuthusamy) [inline page-objects/NewCustomerRegistration-page.js:52]: @karthikeyan7010 Where are you validating credentials here?

**Review by Vigneshmuthusamy — CHANGES_REQUESTED** [review]

**Comment by karthikeyan7010** [comment]
Sorry bro i have completed properly and i share my code with today end of
the day or Monday morning

**Pull request closed without merging**